slides/k8s/kubectlget.md

```diff
@@ -234,12 +315,46 @@ The `kube-system` namespace is used for the control plane.
 
 - the `READY` column indicates the number of containers in each pod
 
-- the pods with a name ending with `-node1` are the master components
-  <br/>
-  (they have been specifically "pinned" to the master node)
+  (1 for most pods, but `weave` has 2, for instance)
```

Comment:
Is it worth clarifying why (worker node count)?

Reply:
We're referring to the number of containers in the pod; in the case of weave, each pod has two containers: one providing the pod network, the other providing network policy implementation (and aptly named weave-npc, for "network policy controller").

I'll merge and make myself a note, and if the slide is not too loaded, I'll add more details!